benchmarks/styled-components/src/components/XStyledTable.js

```diff
@@ -16,7 +16,7 @@ const Cell = styled('div')`
   background: rgba(74, 174, 53, ${props => props.value});
 `
 
-export default ({ table, toPercent }) => (
+const xstyledTable = ({ table, toPercent }) => (
```

Comment:
`export default function XStyledTable`

Reply:
I changed it because there was no visual indication of which table was being rendered on the screen. On line 33 I added a `displayName` property that is accessed later on to display the current component being rendered.

Reply:
I changed this and all mentions to `displayName` to a wrapped div inside each table with the respective title.